system/lib/libc/musl/arch/emscripten/syscall_arch.h

```diff
@@ -55,6 +55,7 @@ int __syscall_munlockall(void);
 int __syscall_mremap(intptr_t old_addr, size_t old_size, size_t new_size, int flags, intptr_t new_addr);
 int __syscall_poll(intptr_t fds, int nfds, int timeout);
 int __syscall_getcwd(intptr_t buf, size_t size);
+const char* __syscall_getcwd_woutargs();
```

Comment:
If possible we should avoid adding a new syscall here.

Reply:
I was getting an unidentified declaration error without adding this function in this class. Any alternate suggestions from your end could help.